modules/react/menu/lib/Menu.tsx

```diff
@@ -40,13 +33,9 @@ export interface MenuProps<T> extends MenuModelConfig<T> {
  *   </Menu.Popper>
  * </Menu>
  */
-export const Menu = createComponent()({
+export const Menu = createContainer()({
```

Comment:
do we need to give it a default element here?

Reply:
`Menu` does not represent an element. All containers are responsible for creating a model context, but not all also represent an element. Tabs is similar - according to the specification, there's no wrapper container element needed for tabs to work. This means the container would not represent a semantic element. We've opted to decrease the amount of elements needed for components.